### PR TITLE
feat(scale): support mirroring

### PR DESCRIPTION
**Pull request by nikitavbv**

### CODE REVIEW

File: src/main/java/com/labs/introtoprogramming/lab4/image/transform/Scale.java [hunk 1]
```diff
@@ -22,13 +22,14 @@ public Scale(double horizontalScale, double verticalScale) {
   public RGBImage applyTo(RGBImage image) {
     int w = image.width();
     int h = image.height();
-    int scaledWidth = (int) (w * horizontalScale);
-    int scaledHeight = (int) (h * verticalScale);
+    int scaledWidth = Math.abs((int) (w * horizontalScale));
+    int scaledHeight = Math.abs((int) (h * verticalScale));
     RGBImage scaledImage = new RGBImage(scaledWidth, scaledHeight);
     double widthRation = (double) (w - 1)
             / (scaledWidth - 1 > 1 ? scaledWidth - 1 : scaledWidth);
```
Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab4/image/transform/Scale.java]: Change here to scalledWidth - 1>= 1 due to bug with scaling picture 2x2 by 1

File: src/main/java/com/labs/introtoprogramming/lab4/image/transform/Scale.java [hunk 2]
```diff
@@ -22,13 +22,14 @@ public Scale(double horizontalScale, double verticalScale) {
...
     double heightRation = (double) (h - 1)
             / (scaledHeight - 1 > 1 ? scaledHeight - 1 : scaledHeight);
```
Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab4/image/transform/Scale.java]: Same as above

File: src/main/java/com/labs/introtoprogramming/lab4/image/transform/Scale.java (hunk 1 above)
Comment (nikitavbv) [reply src/main/java/com/labs/introtoprogramming/lab4/image/transform/Scale.java]: Good catch!

File: src/main/java/com/labs/introtoprogramming/lab4/Main.java [hunk 3]
```diff
@@ -8,6 +8,7 @@
 import com.labs.introtoprogramming.lab4.image.io.bmp.BMPImageWriter;
 import com.labs.introtoprogramming.lab4.image.transform.RGBImageTransformation;
 import com.labs.introtoprogramming.lab4.image.transform.Scale;
+import com.labs.introtoprogramming.lab4.image.transform.SimpleBrightness;
```
Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab4/Main.java]: Remove this import. Brightness is not implemented yet :)

Comment (nikitavbv) [reply src/main/java/com/labs/introtoprogramming/lab4/Main.java]: Sure.

**Review by Brookoko — APPROVED** [review]

**Pull request merged**